packages/shadcn-ui/src/executors/add/add.impl.ts

```diff
@@ -13,7 +13,7 @@ export async function addExecutor(
   const { root } = context.workspace.projects[context.projectName]
 
   return execCommand(buildCommand([
-    'npx shadcn-ui@latest add',
+    `${getPackageManagerCommand().dlx} shadcn-ui@latest add`,
```

Comment:
Could you change this to use `execPackageManagerCommand` from `@nx-extend/core`.

Reply:
See for example the [firebase hosting one](https://github.com/TriPSs/nx-extend/blob/master/packages/firebase-hosting/src/executors/deploy/deploy.impl.ts#L22)

Reply:
That won't work. `execPackageManagerCommand` is using:
```
export function getPackageManagerExecCommand() {
  return getPackageManagerCommand(detectPackageManager()).exec
}
```

which will then run:
```
pnpm exec shadcn-ui@latest add {{component}} --path=src --cwd={{root}}
```
`pnpm` alternative to `npx` is `pnpm dlx`, not `pnpm exec` ([source](https://pnpm.io/feature-comparison)).

Reply:
Do you think we should create `dlxPackageManagerCommand()` in `@nx-extend/core` then?

Reply:
Cool didn't now that something like `npx` existed in the other package managers. Let's update update `getPackageManagerExecCommand` indeed. Good one.